### ISSUE TITLE
Warnings in logstash logs on startup: "deprecated in favor of declaring the version"

### ISSUE THREAD

**Issue by cpliakas** [codekitchen/logstash-input-kinesis#8]
This doesn't seem to affect function at all, it is more of a minor annoyance. On startup we are seeing the following warning in the logstash logs:

```
kinesis plugin is using the 'milestone' method to declare the version of the plugin this method is deprecated in favor of declaring the version inside the gemspec.
```

Thanks for a great plugin.
Chris

**Comment by codekitchen** [comment]
Thanks I've been meaning to fix this for a while. Just released v1.4.2 which removes the `milestone` declaration.

**Comment by cpliakas** [comment]
@codekitchen Awesome! Thanks for the quick turnaround.
